platform/engineering/design-docs/2020-12-10-auto-deploy-cms-build.md

```diff
@@ -247,34 +209,32 @@ There are no new privacy concerns with the auto-deploy as user data is not invol
 
 #### Risks
 
-- Newly added content from CMS without associated transformer schemas could fail the build.
-  - New content introduces the possibility of new discrepancies, which would fail any validations that depend on diffs between the GraphQL and CMS export builds.
-  - Having more content will inflate the runtime of the content-only deploy and could potentially exceed the scheduled interval.
-  - The best defense against this is accounting for null and empty fields in transformers.
+- Having more content will inflate the runtime of the content-only deploy and could potentially exceed the scheduled interval.
 
 ### Work Estimates
 
-- Create a new content-only deploy job for CMS export builds (5 pts)
+- Create a new content-only deploy job (5 pts)
```

Comment:
@theodur This exists already, should just be a matter of updating the function to run on a schedule. @U-DON has more context into that part

Reply:
Yeah that's true, this job already exists. We just need to utilize it for the auto deploy. Just removed this line.